docs/src/ref_impl/example_project.md

```diff
@@ -0,0 +1,36 @@
+<!--
+    Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Example Project
+
+The compartmentalization could be applied to many specific x86_64 computers and laptops with some customization applied to the Ghaf. The best way of the Ghaf customization is using Ghaf templates.
+
+1. Create a template project as described in [Ghaf as Library](../ref_impl/ghaf-based-project.md) section
+2. Adjust your system configuration with accordance to your HW specification. Determine all VIDs and PIDs of the devices that are passed to the VMs
```

Comment:
s/VIDs/vendor IDs (VIDs)/
s/PIDs/product IDs (PIDs)/

Reply:
Discussed this with @jenninikko  - worth adding those to the Glossary section.